app/default/content_routes.py

```diff
@@ -54,3 +57,27 @@ def contact_us():
 def cookie_policy():
     current_app.logger.info("Cookie policy page loaded.")
     return render_template("cookie_policy.html")
+
+
+@content_bp.route("/privacy", methods=["GET"])
+def privacy():
+    current_app.logger.info("Privacy_notice page loaded.")
+    round_data = get_round_data_fail_gracefully(
+        Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID
+    )
+
+    privacy_notice_url = (
+        round_data.privacy_notice
+        if hasattr(round_data, "privacy_notice")
+        else None
+    )
+
+    if privacy_notice_url:
+        return redirect(privacy_notice_url)
+    else:
+        current_app.logger.warning(
+            "No privacy notice configured for fund. Redirecting..."
+        )
+        return redirect(
+            "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice"
+        )
```

Comment:
Could probably leave the else since we're using a return early pattern, would leave out a layer of indentation (personal preference, not a big deal):
```suggestion
    
    current_app.logger.warning(
        "No privacy notice configured for fund. Redirecting..."
    )
    return redirect(
        "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice"
    )
```